geode-protobuf/src/test/java/org/apache/geode/internal/protocol/protobuf/ProtobufShiroAuthenticatorJUnitTest.java

```diff
@@ -41,7 +42,7 @@
   private static final String TEST_USERNAME = "user1";
   private static final String TEST_PASSWORD = "hunter2";
   private ByteArrayInputStream byteArrayInputStream; // initialized with an incoming request in
-                                                     // setUp.
+  // setUp.
```

Comment:
So, does spotless just randomly change how it does formatting?  This doesn't look like something you changed, and presumably it was passing spotlessApply prior to this...so why did this get changed now?  (Not really an issue with your code, just venting)

Reply:
fixed

Reply:
fixed